src/Componenents/Movies.js

```diff
@@ -0,0 +1,33 @@
+import { Component } from "react";
+import CardGroup from 'react-bootstrap/CardGroup';
+import Movie from './Movie';
+
+
+class Movies extends Component {
+
+
+  displayData = (obj) => {
+    let arr = [];
+    for (let i = 0; i < 3; i++) {
+      arr.push(
+        <Movie
+          data={obj.data[i]} />
+      )
+    }
+    return arr;
+  }
+
+  render() {
+
+
+    return (
+      <>
+        <CardGroup className='shadow-sm p-3 mb-5 bg-white rounded'>
+          {this.displayData(this.props.data.data)}
+        </CardGroup>
+      </>
+
```

Comment:
Really nice work, when you have time, I would clean up spacing for this file. 
1 line between top level blocks is sufficient. Vsual separation between separate things is good, but 1 space is usually enough between functions, etc. 👍